### ISSUE TITLE
Try to get Field flow basic configuration working

### ISSUE THREAD

**Issue by gsabatin** [wet-boew/wet-boew#8454]
I can't seem to get the checkbox example working on my page. Attached is my HTML page (note: this page sits behind a login).

I'm not sure if I am initializing the data-ajax stuff correctly.
[Describing Your Skills, Experience, and Accomplishments with STAR Statements.zip](https://github.com/wet-boew/wet-boew/files/2271643/Describing.Your.Skills.Experience.and.Accomplishments.with.STAR.Statements.zip)

**Comment by duboisp** [comment]
Are you using the GCWeb template?

Did you see any error in the console?

Is those file exist? 
* ajax/ajax-1.html
* ajax/ajax-2.html
* ajax/ajax-3.html
* ajax/ajax-4.html
* ajax/ajax-5.html

**Comment by gsabatin** [comment]
I’m not using the GCWeb template, just the base. Is that a requirement? Can I add the field flow configuration to the base?

Greg Sabatine
Manager, Online Technologies
Open Learning and Educational Support
University of Guelph
519.824.4120 ext. 56042

**Comment by nschonni** [comment]
Fieldflow is only in GCWeb, it is not part of WET

**Comment by duboisp** [comment]
> I’m not using the GCWeb template, just the base. Is that a requirement?

Yes, it is currently because fieldflow are only implemented into the GCWeb template.

> Can I add the field flow configuration to the base?

Yes you can add it in your own wet-boew instance. It should be feasible by adding the following javascript file after the wet-boew.js script in your page. https://github.com/wet-boew/GCWeb/blob/master/src/plugins/fieldflow/fieldflow.js

**Comment by gsabatin** [comment]
Thanks, I’ll give it a try.

Greg Sabatine
Manager, Online Technologies
Open Learning and Educational Support
University of Guelph
519.824.4120 ext. 56042

**Comment by gsabatin** [comment]
It seems to be working, except it’s forcing me to select all of the items when I use the checkbox UI.

Greg Sabatine
Manager, Online Technologies
Open Learning and Educational Support
University of Guelph
519.824.4120 ext. 56042

**Comment by duboisp** [comment]
Do you have a link where we can test you implementation?

**Comment by gsabatin** [comment]
Hi, try this link.

https://courselink.uoguelph.ca/shared/css/WET.4.0.28/Describing%20Your%20Skills,%20Experience,%20and%20Accomplishments%20with%20STAR%20Statements.html


Greg Sabatine
Manager, Online Technologies
Open Learning and Educational Support
University of Guelph
519.824.4120 ext. 56042

**Comment by duboisp** [comment]
Update wet-boew.js file with the latest version, 4.0.28.1 - https://github.com/wet-boew/wet-boew-dist/blob/v4.0.28.1/js/wet-boew.min.js

Try again and let me know if it is working.

**Comment by duboisp** [comment]
Another option will be to make Field flow optional like:

```html
<div class="wb-fieldflow" data-wb-fieldflow='{"renderas":"checkbox", "isoptional": true}'>
```

**Comment by gsabatin** [comment]
Updating the wet-boew js file didn’t work but making it optional did the trick. Thanks!


Greg Sabatine
Manager, Online Technologies
Open Learning and Educational Support
University of Guelph
519.824.4120 ext. 56042
